plugins/obs-ffmpeg/texture-amf.cpp

```diff
@@ -1171,7 +1171,7 @@ static bool amf_avc_init(void *data, obs_data_t *settings)
 
 	info("settings:\n"
 	     "\trate_control: %s\n"
-	     "\tbitrate:      %d\n"
+	     "\tbitrate:      %d kbps\n"
```

Comment:
Since none of the other items have units, and neither do any of the other encoders, I'd omit that here for now. Eventually we should perhaps revise the logging for this entirely and add units where appropiate.

Reply:
I was actually hoping to PR a change to all of the other encoders to add kbps.

Apple VT already has it, and I would posit that knowing the unit can only be helpful.

Do you think it should be in a separate PR (per encoder, or all together)?

Reply:
Seeing as this breaks the logAnalyzer, I'll just remove it :)